### PR TITLE
2119 there are a few tracts places where the values over the 90th percentile are not showing as blue

### PR DESCRIPTION
**Pull request by sampowers-usds**
The main change here is in the score/constants.py file, adjusting IS_ET to pull on NON_NATURAL_PCTILE_THRESHOLD as opposed to NON_NATURAL_LOW_INCOME_FIELD_NAME. Also adding the ipython notebook I used to investigate to the ipython folder

### CODE REVIEW

File: data/data-pipeline/data_pipeline/score/score_narwhal.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import data_pipeline.score.field_names as field_names
 import numpy as np
 import pandas as pd
-from data_pipeline.score.score import Score
+from data_pipeline.score.score import Score # this just adds the framework of the score class
```
Comment (vim-usds) [inline data/data-pipeline/data_pipeline/score/score_narwhal.py:7]: Not sure if "above line" comments are preferred over in-line comments?

File: data/data-pipeline/data_pipeline/score/score_narwhal.py [hunk 2]
```diff
@@ -14,8 +14,8 @@
 class ScoreNarwhal(Score):
     """Score N, aka Narwhal."""
 
-    LOW_INCOME_THRESHOLD: float = 0.65
-    MAX_COLLEGE_ATTENDANCE_THRESHOLD: float = 0.20
+    LOW_INCOME_THRESHOLD: float = 0.65               # this is the low income threshold that gets compared against the other indicators. It is a percentile rank score
```
Comment (vim-usds) [inline data/data-pipeline/data_pipeline/score/score_narwhal.py:17]: these comments are super helpful!

**Comment by vim-usds** [comment]
@sampowers-usds - seems as tho maybe some of the smoke tests are failing. I think it may have to do with the donut calculations?

**Comment by vim-usds** [comment]
This looks great!!

It seems that each of the original tracts outlined by @KameronKerger for the lack of green space not being highlighted when over 90th percentile seems functional!

- [ ] originally [failing tract on prod](https://screeningtool.geoplatform.gov/en/#15.06/34.02823/-118.31329) is now [passing on staging](https://screeningtool.geoplatform.gov/en?flags=stage_hash=2160/2d31365ea6bbd7719accbb660157faba980f97a3#15.06/34.02823/-118.31329)
- [ ] originally [failing tract on prod](https://screeningtool.geoplatform.gov/en/#15.05/34.09937/-118.09367) is now [passing on staging](https://screeningtool.geoplatform.gov/en?flags=stage_hash=2160/2d31365ea6bbd7719accbb660157faba980f97a3#14.61/34.09937/-118.09367)
- [ ] originally [failing tract on prod](https://screeningtool.geoplatform.gov/en/#15.63/34.096116/-118.130794) is now [passing on staging](https://screeningtool.geoplatform.gov/en?flags=stage_hash=2160/2d31365ea6bbd7719accbb660157faba980f97a3#15.63/34.096116/-118.130794)

Out of curiosity, were the total number of tracks designated as disadvantaged changed by any amount due to the fix?

**Comment by vim-usds** [comment]
@sampowers-usds - the[ original ticket](https://github.com/usds/justice40-tool/issues/2119) also calls out some issues around the lead paint indicator. I'm not sure I understand the problem and if others are feeling the same, maybe we can create a new ticket to identify this, as this PR seems to fix the lack of green space issue.

**Comment by travis-newby** [comment]
> * I compared all of the census tracts. There are 51 tracts with at least one score difference. Please examine the logs or run the score comparison locally to view them all.

These are the same deltas we've seen in main

**Comment by vim-usds** [comment]
All three problem tracts still have the lack of green space field fixed!

1. https://screeningtool.geoplatform.gov/en?flags=stage_hash=2160/f8d5ec7e00ca380519398b8f6663806383106e0f#15.06/34.02823/-118.31329
2. https://screeningtool.geoplatform.gov/en?flags=stage_hash=2160/f8d5ec7e00ca380519398b8f6663806383106e0f#14.61/34.09937/-118.09367
3. https://screeningtool.geoplatform.gov/en?flags=stage_hash=2160/f8d5ec7e00ca380519398b8f6663806383106e0f#14.72/34.09612/-118.13079

**Review by vim-usds — APPROVED** [review]
Great work team!

**Pull request merged**